clients/admin-ui/src/pages/management/cors-configuration.tsx

```diff
@@ -0,0 +1,172 @@
+import { AddIcon } from "@chakra-ui/icons";
+import {
+  Box,
+  Button,
+  DeleteIcon,
+  Flex,
+  Heading,
+  IconButton,
+  Link,
+  Text,
+  useToast,
+} from "@fidesui/react";
+import { SerializedError } from "@reduxjs/toolkit";
+import { FetchBaseQueryError } from "@reduxjs/toolkit/dist/query";
+import { FieldArray, Form, Formik, FormikHelpers } from "formik";
+import type { NextPage } from "next";
+
+import { useAppSelector } from "~/app/hooks";
+import FormSection from "~/features/common/form/FormSection";
+import { CustomTextInput } from "~/features/common/form/inputs";
+import { getErrorMessage, isErrorResult } from "~/features/common/helpers";
+import Layout from "~/features/common/Layout";
+import { errorToastParams, successToastParams } from "~/features/common/toast";
+import {
+  selectCORSDomains,
+  useCreateConfigurationSettingsMutation,
+  useGetConfigurationSettingsQuery,
+} from "~/features/privacy-requests/privacy-requests.slice";
+
+type FormValues = {
+  domains: string[];
+};
+
+const CORSConfigurationPage: NextPage = () => {
+  const { isLoading: isLoadingGetQuery } = useGetConfigurationSettingsQuery();
+  const corsDomains = useAppSelector(selectCORSDomains());
+  const [patchConfigSettingsTrigger, { isLoading: isLoadingPatchMutation }] =
+    useCreateConfigurationSettingsMutation();
+
+  const toast = useToast();
+  if (isLoadingGetQuery) {
+    return <div>loading</div>;
+  }
+
+  const handleSubmit = async (
+    values: FormValues,
+    formikHelpers: FormikHelpers<FormValues>
+  ) => {
+    const handleResult = (
+      result: { data: {} } | { error: FetchBaseQueryError | SerializedError }
+    ) => {
+      toast.closeAll();
+      if (isErrorResult(result)) {
+        const errorMsg = getErrorMessage(
+          result.error,
+          `An unexpected error occurred while saving CORS domains. Please try again.`
+        );
+        toast(errorToastParams(errorMsg));
+      } else {
+        toast(successToastParams("CORS domains saved successfully"));
+        // Reset state such that isDirty will be checked again before next save
+        formikHelpers.resetForm({ values });
+      }
+    };
+
+    const payload = {
+      security: {
+        cors_origins: values.domains,
+      },
+    };
+    const result = await patchConfigSettingsTrigger(payload);
+
+    handleResult(result);
+  };
+
+  return (
+    <Layout title="CORS Configuration">
+      <Box data-testid="cors-configuration">
+        <Heading marginBottom={4} fontSize="2xl">
+          CORS Configuration
+        </Heading>
+        <Box maxWidth="600px">
+          <Text marginBottom={2} fontSize="md">
+            Add your CORS domains below
+          </Text>
+          <Text mb={10} fontSize="sm">
+            Please visit{" "}
+            <Link
+              color="complimentary.500"
+              href="https://docs.ethyca.com"
+              isExternal
+            >
+              docs.ethyca.com
+            </Link>{" "}
```

Comment:
there's actually a [DocsLink component](https://github.com/ethyca/fides/blob/cors-domain-config-api/clients/admin-ui/src/features/common/DocsLink.tsx) we could use here